index.js

```diff
@@ -4,7 +4,7 @@ var program = require('commander'),
   request = require('request');
 
 program
-  .arguments('<slng>')
+  .arguments('<search_string>')
```

Comment:
We don't need this to be snake or kebab case. we can just leave it as plain text `search string`